firefox_puppeteer/api/utils.py

```diff
@@ -10,6 +10,23 @@
 class Utils(BaseLib):
     """Low-level access to utility actions."""
 
+    def remove_perms(self, host, permission):
+        """Remove permission for web host.
+
+        Permissions include safe-browsing, install, geo, and others described here:
+        https://dxr.mozilla.org/mozilla-central/source/browser/modules/SitePermissions.jsm#144
+        and elsewhere.
+
+        :param host: The web host whose permission will be removed.
+        :param permission: The type of permission to be removed.
+        """
+
+        with self.marionette.using_context('chrome'):
+            self.marionette.execute_script("""
+              Cu.import("resource://gre/modules/Services.jsm");
```

Comment:
Please don't use the shortcut for `Components.utils`, given that it will fail if you call this method if not a brower window is currently selected.